src/utils/src/FileLogger.c

```diff
@@ -0,0 +1,250 @@
+/**
+ * Kinesis Video Producer File based logger
+ */
+#define LOG_CLASS "FileLogger"
+#include "Include_i.h"
+
+static PFileLogger gFileLogger = NULL;
```

Comment:
Important! Make sure remove static keyword here to ensure a single instance for the entire process address space